src/node/media_stream_destination.rs

```diff
@@ -96,15 +102,13 @@ impl MediaStreamAudioDestinationNode {
         })
     }
 
-    /// A [`MediaStream`] iterator producing audio buffers with the same number of channels as the
-    /// node itself
+    /// A [`MediaStream`] producing audio buffers with the same number of channels as the node
+    /// itself
     ///
-    /// Note that while you can call this function multiple times and poll all iterators concurrently,
+    /// Note that while you can call this function multiple times and poll all streams concurrently,
     /// this could lead to unexpected behavior as the buffers will only be offered once.
-    pub fn stream(&self) -> impl MediaStream {
-        AudioDestinationNodeStream {
-            receiver: self.receiver.clone(),
-        }
+    pub fn stream(&self) -> &MediaStream {
```

Comment:
"as the buffers will only be offered once." is this still true ? it seemed to me that `MediaStreamTrackIter.position` was there to prevent that, but maybe I misunderstood something